Source/WebCore/css/SelectorCheckerTestFunctions.h

```diff
@@ -37,6 +37,7 @@
 #include "HTMLOptionElement.h"
 #include "InspectorInstrumentation.h"
 #include "Page.h"
+#include "PopoverData.h"
```

Comment:
Always annoying to add any include to any `.h`.

Reply:
Indeed, but I had to draw a line or the patch would get too big. :)